ux.symfony.com/assets/styles/components/_PackageBox.scss

```diff
@@ -120,6 +120,10 @@
   letter-spacing: -1px;
   color: #0A0A0A;
   text-decoration: none;
+  
+  [data-bs-theme="dark"] & {
+    filter: invert(1);
+  }
 }
```

Comment:
```
color: var(--bs-body-color);
text-decoration: none;
```

Simpler no ?

Reply:
"Depends" ;) If you mean to change it on line 121, then it will be changed to a light theme.

I followed already used code from: https://github.com/symfony/ux/blob/2.x/ux.symfony.com/assets/styles/sections/_header.scss#L39

Reply:
Because i wanted to invert all the colors, but if you look on other CSS properties, i tried to use CSS vars as much as possible because it's easier to read/maintain.. 

But i won't go against your suggestion :)) Iit's a good catch and a really nice attention to open this PR, and i'm really glad you did :)